### ISSUE TITLE
sh copy_local.sh copies to the wrong folder

### ISSUE THREAD

**Issue by marin-bratanov** [telerik/docs-seed#49]
**Describe the bug**

Running the `sh copy_local.sh` command from the seed repo towards the content repo causes it to create a folder under it and place the new files there, instead of at the provided path. The new folder is the provided path stripped of backslashes and semicolons.

Thus, attempting to build docs locally causes the following error where the last line shows the path distortion:

```
Marin@Marin-Laptop MINGW64 /c/Telerik/Git/ajax-docs (master)
$ bundle exec jekyll build
Configuration file: C:/Telerik/Git/ajax-docs/_config.yml
            Source: C:/Telerik/Git/ajax-docs
       Destination: C:/Telerik/Git/ajax-docs/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
     Build Warning: Layout 'documentation' requested in web.config does not exist.
jekyll 3.3.1 | Error:  [prev_next plugin] Comparing the pages /telerikgitajax-docs and sitemap.xml failed
```

**To Reproduce**

1. Get latest of the seed repo as of 20 Feb 2019 (the date of testing)
1. clone the ajax-docs content repo
1. copy the files from the seed repo to the ajax repo

**Expected behavior**

working docs, the necessary files copied at the provided path.

**Desktop (please complete the following information):**
 - OS: [Windows 10]

**Comment by imtodor** [comment]
Are you passing the path argument to the ‘sh copy_local.sh’ with quotes? (e.g. “d:\work\ajax-docs”)

The builds are using that bash file as well and they are all green.

**Comment by imtodor** [comment]
The AJAX-DOCS build is also passing, so I'm closing the issue.

**Comment by marin-bratanov** [comment]
I am attaching here artifacts that show how this happens for a brand new repo. It seems something somewhere is hardcoded for that `aspnetajax-docs` folder because it is present in the console output of the bash script even if the arguments have nothing to do with ajax.
[wrong-folder-path-copied-artifacts.zip](https://github.com/telerik/docs-seed/files/2898181/wrong-folder-path-copied-artifacts.zip)
